<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="80495387-0cd8-4043-83bf-569e5b566894" name="joe@AGH 2012-01-29 09:40:58" runUser="AGH\joe" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="e0f90ab9-fee6-4fc8-a9ad-cd5e0507f10d">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment enabled="false" runDeploymentRoot="joe_AGH 2012-01-29 09_40_58">
      <DeploymentItem filename="C:\Users\joe\Documents\Visual Studio 2010\Projects\KPFF_C#\KPFF_Csharp_Converted\KPFF.Web\bin\KPFF.PMP.dll" />
    </Deployment>
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2012-01-29T09:40:58.0543434-08:00" queuing="2012-01-29T09:40:58.1483488-08:00" start="2012-01-29T09:40:58.3213587-08:00" finish="2012-01-29T09:41:00.9375083-08:00" />
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" error="0" failed="1" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="GetGroupByID" storage="c:\users\joe\documents\visual studio 2010\projects\kpff_c#\kpff_csharp_converted\pmp.test\bin\debug\pmp.test.dll" id="40b9dffa-65a9-9769-6118-a28458db381d">
      <Execution id="95a4677b-07f5-4711-a685-f275d8057f37" />
      <TestMethod codeBase="C:/Users/joe/Documents/Visual Studio 2010/Projects/KPFF_C#/KPFF_Csharp_Converted/PMP.Test/bin/Debug/PMP.Test.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="PMP.Test.EngineerGroupsTest, PMP.Test, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="GetGroupByID" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="40b9dffa-65a9-9769-6118-a28458db381d" executionId="95a4677b-07f5-4711-a685-f275d8057f37" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="95a4677b-07f5-4711-a685-f275d8057f37" testId="40b9dffa-65a9-9769-6118-a28458db381d" testName="GetGroupByID" computerName="AGH" duration="00:00:00.2714444" startTime="2012-01-29T09:40:58.5143697-08:00" endTime="2012-01-29T09:40:59.2974145-08:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="95a4677b-07f5-4711-a685-f275d8057f37">
      <Output>
        <ErrorInfo>
          <Message>Test method PMP.Test.EngineerGroupsTest.GetGroupByID threw exception: 
System.ApplicationException: An error occured in GetDataReaderByStoredProcedure. ---&gt; System.InvalidOperationException: There is already an open DataReader associated with this Command which must be closed first.</Message>
          <StackTrace>    at System.Data.SqlClient.SqlInternalConnectionTds.ValidateConnectionForExecute(SqlCommand command)
   at System.Data.SqlClient.SqlConnection.ValidateConnectionForExecute(String method, SqlCommand command)
   at System.Data.SqlClient.SqlCommand.ValidateCommand(String method, Boolean async)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, DbAsyncResult result)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior)
   at KPFF.PMP.Entities.DABase.GetDataReaderByStoredProcedure(String procedureName, Dictionary`2 params, SqlConnection con) in C:\Users\joe\Documents\Visual Studio 2010\Projects\KPFF_C#\KPFF_Csharp_Converted\KPFF.Web\Entities\DABase.cs:line 70
 --- End of inner exception stack trace ---
    at KPFF.PMP.Entities.DABase.GetDataReaderByStoredProcedure(String procedureName, Dictionary`2 params, SqlConnection con) in C:\Users\joe\Documents\Visual Studio 2010\Projects\KPFF_C#\KPFF_Csharp_Converted\KPFF.Web\Entities\DABase.cs:line 74
   at KPFF.PMP.Entities.EngineerGroup.GetById(Int32 groupId) in C:\Users\joe\Documents\Visual Studio 2010\Projects\KPFF_C#\KPFF_Csharp_Converted\KPFF.Web\Entities\EngineerGroup.cs:line 98
   at PMP.Test.EngineerGroupsTest.GetGroupByID() in C:\Users\joe\Documents\Visual Studio 2010\Projects\KPFF_C#\KPFF_Csharp_Converted\PMP.Test\EngineerGroupsTest.cs:line 31
</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>